src/script/lua_api/l_object.cpp

```diff
@@ -1181,6 +1181,29 @@ int ObjectRef::l_get_player_control_bits(lua_State *L)
 	return 1;
 }
 
+
+// set_minimap_mode(self, id)
+int ObjectRef::l_set_minimap_mode(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	Player *player = getplayer(ref);
+	if (player == NULL)
+		return 0;
+	// TODO
+}
```

Comment:
I have a warning during compilation: `warning: control reaches end of non-void function [-Wreturn-type]`